include/oneapi/mkl/blas.hxx

```diff
@@ -263,19 +263,22 @@ static inline void gemm(cl::sycl::queue &queue, transpose transa, transpose tran
 }
 
 static inline void gemm(cl::sycl::queue &queue, transpose transa, transpose transb, std::int64_t m,
-                        std::int64_t n, std::int64_t k, half alpha, cl::sycl::buffer<half, 1> &a,
-                        std::int64_t lda, cl::sycl::buffer<half, 1> &b, std::int64_t ldb, half beta,
-                        cl::sycl::buffer<half, 1> &c, std::int64_t ldc) {
+                        std::int64_t n, std::int64_t k, cl::sycl::half alpha,
+                        cl::sycl::buffer<cl::sycl::half, 1> &a, std::int64_t lda,
```

Comment:
Initially, we were using a local implementation for half precision. We will be moving to sycl::half soon. You can remove these changes from PR I think. we would like to do the changes globally and validate them with CI.